testnets/nilliontestnet/assetlist.json

```diff
@@ -0,0 +1,34 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "nilliontestnet",
+  "assets": [
+    {
+      "description": "NIL - the native token of Nillion",
+      "denom_units": [
+        {
+          "denom": "unil",
+          "exponent": 0,
+          "aliases": []
+        },
+        {
+          "denom": "nil",
+          "exponent": 6,
+          "aliases": []
+        }
+      ],
+      "base": "unil",
+      "name": "Nillion",
+      "display": "nil",
+      "symbol": "NIL",
+      "logo_URIs": {
```

Comment:
Please omit logo_URIs (but keep `images`)

Reply:
Fixed

Reply:
Please delete the logo_URIs property

Reply:
Removed